README.md

```diff
@@ -212,6 +212,47 @@ try {
 }
 ```
 
+### Webhooks
+
+Recurly can send webhooks to any publicly accessible server.
+When an event in Recurly triggers a webhook (e.g., an account is opened),
+Recurly will attempt to send this notification to the endpoint(s) you specify.
+You can specify up to 10 endpoints through the application. All notifications will
+be sent to all configured endpoints for your site.
```

Comment:
The line wrapping on this is a little odd but I guess GitHub will turn it into a single <p> tag right?

Reply:
Yes, normalized it some though.